pika/adapters/blocking_connection.py

```diff
@@ -1496,8 +1500,8 @@ def add_on_return_callback(self, callback):
                         callback, self, method, properties, body))))
 
     def basic_consume(self,
-                      consumer_callback,
-                      queue,
+                      callback,
+                      queue='',
```

Comment:
Why is queue param defaulted to `''` in this commit? The queue name is not optional, right?

Reply:
I wanted to make sure all `queue` args were the same and defaulted to the empty string, which is supported by RabbitMQ. Basically, the empty string means "use the last auto-named queue". I don't particularly care for this default, to be honest.